src/client/src/apps/MainRoute/widgets/spotTile/components/Tile/TileBusinessLogic.ts

```diff
@@ -12,10 +12,10 @@ const MIN_RFQ_VALUE = 10000000
 
 // Utils
 export const getFormattedValue = (value: number | string) => numeral(value).format(NUMERAL_FORMAT)
-export const getDefaultNotionalValue = (currencyPair: CurrencyPair) =>
+export const getDefaultNotionalValue = (currencyPair: CurrencyPair | undefined) =>
   // This is to simply to have one Tile showing RFQ prompt on page load
   // check JIRA ticket ARTP-532
-  currencyPair.symbol === 'NZDUSD' ? MIN_RFQ_VALUE : DEFAULT_NOTIONAL_VALUE
```

Comment:
this is where exception was happening. If the store is not yet initialized (no currencies yet), then we `currencyPair` comes as `undefined` and exception happens